enterprise/cluster/src/main/java/org/neo4j/cluster/client/Clusters.java

```diff
@@ -203,14 +201,7 @@ public boolean isFullHaMember()
 
         private static String localhost()
         {
-            try
-            {
-                return InetAddress.getLocalHost().getHostAddress();
-            }
-            catch ( UnknownHostException e )
-            {
-                throw new RuntimeException( e );
-            }
+            return "127.0.0.1";
```

Comment:
getHostAddress can return 127.0.0.1, 127.0.1.1 or the host IP. Setting it to a fixed 127.0.0.1 makes sure we avoid issues where different instances get different results. And this is only for testing anyway, so does not impact customers.